spark-publish/index.js

```diff
@@ -15,6 +15,7 @@ export const publish = async ({
       id,
       spark_version,
       zinnia_version,
+      -- wallet_address,
```

Comment:
why is this commented out?

Reply:
Oops, I wanted to see if there is any failing test and then I forgot to uncomment 🙈

Reply:
Fixed. I also added a test to verify that we are correctly converting the address from Filecoin to Ethereum format.